bin/verilator

```diff
@@ -363,6 +363,7 @@ arguments.
     --pins-bv <bits>            Specify types for top level ports
     --pins-sc-uint              Specify types for top level ports
     --pins-sc-biguint           Specify types for top level ports
+    --pins-sc-biguint-nolimit   Specify types for top level ports
```

Comment:
I think this needs to be updated, you used --pins-limit, but I think what you had here makes more logical sense without the "no" that is "--pins-sc-buguint-limit", so please update the flag to that, then I think it's good to merge.  Thanks.